build.gradle

```diff
@@ -140,6 +143,15 @@ allprojects {
                 "experimental:kdoc-wrapping",
         ]
     }
+
+    detekt {
+        // preconfigure defaults
+        buildUponDefaultConfig = true
+        // activate all available (even unstable) rules.
+        allRules = false
```

Comment:
we can generate a baseline if we want to enable more rules without having to fix them https://detekt.dev/docs/introduction/baseline/

Reply:
Yes, thanks, I have seen that. Good to prevent new issues. I would prefer to enable the disabled rules one by one and fix the existing issues though.